mainpage.dox

```diff
@@ -121,17 +121,25 @@
  * as a dependency. For instance, all the demos link to it externally via
  * <a href="https://cdnjs.com/">cdnjs.com</a>.
  *
- * \note The current \c janus.js library makes use of jQuery (http://jquery.com/)
- * as a support. A version without that dependency is available as
- * \c janus.nojquery.js in the same folder, so you can use that one instead
- * if required by your web application: all the documentation related to
- * \c janus.js applies to \c janus.nojquery.js as well (including the
- * dependency on the webrtc-adapter shim).
+ * \note The current \c janus.js library allows you to provide custom implementations of
+ * certain dependencies, in order to make it easier to integrate with other JavaScript
+ * libraries and frameworks. Using this feature you can ensure \c janus.js does not (implicitly)
+ * depend on certain global variables. Two implementations are included in \c janus.js itself:
+ *
+ *  1. \c Janus.useDefaultDependencies which relies on native browser APIs,
+ *     which in turn require somewhat more modern browsers
+ *  2. \c Janus.useOldDependencies which uses jQuery (http://jquery.com/) instead,
+ *     and should provide equivalent behaviour to previous versions of \c janus.js
+ *
+ * By default \c Janus.useDefaultDependencies will be used, but you can override this
+ * when initialising the Janus library and pass a custom dependencies object instead.
+ * For details, refer to \ref js-dependencies the documentation on working with custom
```

Comment:
When you use `\ref anchor`, Doxygen inserts the section title inline. This means that, since in this case the title is "Working with custom janus.js dependencies", it's better to use something like:

    For details, refer to the documentation on \ref js-dependencies about...

as your current text at the moment is rendered like this when compiled:

    For details, refer to Working with custom janus.js dependencies the documentation